simulator/src/main/java/com/github/benmanes/caffeine/cache/simulator/policy/linked/LinkedPolicy.java

```diff
@@ -240,8 +264,14 @@ public void moveToTail() {
     public String toString() {
       return MoreObjects.toStringHelper(this)
           .add("key", key)
+          .add("weight", weight)
           .add("marked", marked)
           .toString();
     }
   }
+  
+  /** Cantor pairing function. */
+  private long hashCode(long key, int weight) {
```

Comment:
fyi, `Objects.hashCode(...)` is a convenience for these cases

Reply:
I tried it, but it has (many?) collisions.